_data/projects/keploy.yml

```
@@ -0,0 +1,13 @@
+name: Keploy
+desc: Keploy is a functional testing toolkit for developers. It generates E2E tests for APIs (KTests) along with mocks or stubs(KMocks) by recording real API calls. KTests can be imported as mocks for consumers and vice-versa.
+site: https://keploy.io/
+tags:
+  - api-testing
+  - mock-generation
+  - go
+  - java
+  - typescript
+  - automated-testing
+upforgrabs:
+  name: good first issue
```

This needs to match the label name exactly

```suggestion
  name: Good First Issue
```